functions/setupApi.js

```diff
@@ -77,12 +77,14 @@ const setupApi = ({ autopilotapikey, middlewares = [] }) => {
   async function subscribe(request, response) {
     const email = request && request.body && request.body.email
     const pathname = request && request.body && request.body.pathname
+    const city = request && request.body && request.body.city
     await postToAutopilot(`/contact`, {
       contact: {
         Email: email,
         LeadSource: 'Footer contact form',
```

Comment:
I think here we should set the LeadSoure to `pathname`. I know this was already there, it's a legacy from when you could only sign up from the footer.